addons/website_sale/views/res_config_settings_views.xml

```diff
@@ -6,8 +6,8 @@
         <field name="model">res.config.settings</field>
         <field name="inherit_id" ref="account.res_config_settings_view_form"/>
         <field name="arch" type="xml">
-            <xpath expr="//button[@name='action_update_terms']" position="replace">
-                <div class="mt8" attrs="{'invisible': [('terms_type', '!=', 'html')]}" groups="website.group_website_designer">
+            <xpath expr="//button[@name='action_update_terms']" position="after">
+                <div class="mt8" attrs="{'invisible': [('terms_type', '!=', 'html')]}" groups="website.group_website_designer" invisible="1">
```

Comment:
This is ok only if we make `website_sale` depends on `website_account`.
Else, if we upgrade the module `website_sale` (without having `website_account` installed), we will endup with this being invisible with e-commerce.

Reply:
I don't think so because website_sale depends on account and website so it will trigger the auto_install for website_account, correct me if I am wrong

Reply:
Indeed, I think you are right

Reply:
I tested locally, intalled only website_sale and it worked, it auto_intalled website_account

Reply:
ah I should test it with upgrade too let me check